docs/installation/post-install.rst

```diff
@@ -26,12 +26,12 @@ path for the partitions to use:
    apiVersion: storage.metalk8s.scality.com/v1alpha1
    kind: Volume
    metadata:
-     name: <node_name>-prometheus
+     name: bootstrap-prometheus
    spec:
      nodeName: <node_name>
      storageClassName: metalk8s-prometheus
-     rawBlockDevice:  # Choose a device with at least 10GiB capacity
-       devicePath: <device_path>
+     sparseLoopDevice:
```

Comment:
No, not at all. While yes this may *work*, it's definitely *not* what we want people to do in production setups.

**We can not give copy-paste instructions**. People have to *understand* what they are doing and *make decisions*, adapting certain things (like these templates) to any such decisions they made.